alonet/panoptic/callbacks.py

```diff
@@ -0,0 +1,46 @@
+import aloscene
```

Comment:
we can probably rename panoptic to detr_panoptic

Reply:
Ok; I got it